std/conv.d

```diff
@@ -147,20 +147,16 @@ Conversions from string _to numeric types differ from the C equivalents
 `atoi()` and `atol()` by checking for overflow and not allowing whitespace.
 
 For conversion of strings _to signed types, the grammar recognized is:
-<pre>
-$(I Integer): $(I Sign UnsignedInteger)
+<pre>$(I Integer): $(I Sign UnsignedInteger)
```

Comment:
There is [`$(PRE`](https://github.com/dlang/dlang.org/blob/master/html.ddoc#L39) as well ;-)